### PR TITLE
Fix shellcheck

### PR DESCRIPTION
**Pull request by kiblik**
Fix for https://github.com/DefectDojo/django-DefectDojo/pull/9119#issuecomment-1850996614

### CODE REVIEW

**Comment by kiblik** [comment]
`shellcheck` is maybe fixed but opened couple of issues now

**Review by cneill — COMMENTED** [review]
One minor comment, but otherwise looks good. Thank you for tackling this! I should've done so when I first added the `shellcheck` job 😅

File: docker/entrypoint-unit-tests-devDocker.sh [hunk 1]
```diff
@@ -29,8 +29,8 @@ You made changes to the REST API without applying the correct schema annotations
 These schema annotations are needed to allow for the correct generation of
 the OpenAPI (v3) schema's and documentation.
 
-Review the warnings generated by drf-spectacular and see `dojo/api_v2/views.py`
-and/or `dojo/api_v2/serializers.py`.
+Review the warnings generated by drf-spectacular and see $(dojo/api_v2/views.py)
+and/or $(dojo/api_v2/serializers.py).
```
[the remark this answers is not in the document]
Comment (kiblik) [reply docker/entrypoint-unit-tests-devDocker.sh]: :) These appeared there because of auto-suggestion and I haven't noticed that. Thank you

**Review by cneill — APPROVED** [review]
Thanks!

**Review by mtesauro — APPROVED** [review]
Approved

**Review by Maffooch — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Pull request merged**